AuthenticatorShared/Core/Platform/Utilities/ExternalLinksConstants.swift

```diff
@@ -30,6 +30,15 @@ enum ExternalLinksConstants {
     /// A link to Bitwarden's import items help webpage.
     static let importItems = URL(string: "http://bitwarden.com/help/import-data/")!
 
+    /// A link to the password manager app within the app store.
+    static let passwordManagerLink = URL(string: "https://itunes.apple.com/app/id1137397744?mt=8")!
+
+    /// The url scheme used by the password manager app
+    static let passwordManagerScheme = URL(string: "bitwarden://")!
+
+    /// A deeplink used by the password manager app to open the options menu.
+    static let passwordManagerSettings = URL(string: "bitwarden://sync_authenticator?options=true")!
+
```

Comment:
🤔 How would this work in Betas? Like **Authenticator Beta** with **Password manager Beta** apps? I'd guess the scheme would change. Maybe it's outside of the scope of this PR, but raising it just in case.

Reply:
It doesn't appear that the scheme changes currently: https://github.com/bitwarden/ios/blob/main/Bitwarden/Application/Support/Info.plist#L74

So I'm thinking there's no need to change this.

However, there is a discussion currently happening regarding the beta builds and the sharing between app groups, so this might be something to keep in mind based on what comes out of that. I'd say probably fine for this PR, but something to keep in mind when we add more configuration files to set up the app groups (which we're in the process of doing).